hack/openstack/test-manifests.sh

```diff
@@ -12,6 +12,10 @@ declare \
 	external_network='' \
 	compute_flavor=''
 
+# Let install-config describe a configuration that is incompatible with the
+# target CI infrastructure
+export OPENSHFIT_INSTALL_SKIP_PREFLIGHT_VALIDATIONS=1
```

Comment:
Perhaps we should remove this export from https://github.com/openshift/release/pull/13950 if you're adding it to the `test-manifests.sh` script directly?

Reply:
done :)